src/main/java/com/mwt/explorers/MwtExplorer.java

```diff
@@ -31,14 +31,14 @@ public MwtExplorer(String appId, Recorder<T> recorder) {
    *
    * @return The chosen action
    */
-  int chooseAction(Explorer<T> explorer, String uniqueKey, T context) {
+  public int chooseAction(Explorer<T> explorer, String uniqueKey, T context) {
     long seed = uniqueKey.hashCode();
```

Comment:
@lhoang29 same thing here for the hash. Don't think that C# and Java will have the same hash function.